Gemfile.lock

```diff
@@ -37,22 +38,23 @@ GEM
       rspec-core (~> 3.9.0)
       rspec-expectations (~> 3.9.0)
       rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.0)
-      rspec-support (~> 3.9.0)
-    rspec-expectations (3.9.0)
+    rspec-core (3.9.2)
```

Comment:
### [rspec-core](https://github.com/rspec/rspec-core)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.9.0 → 3.9.2

[_[change-log](https://github.com/rspec/rspec-core/blob/v3.9.2/Changelog.md), [source-code](https://github.com/rspec/rspec-core)_]





<details>
<summary>Commits</summary>

A change of **25** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-core/compare/v3.9.0...v3.9.2).

These are the first 10 commits:
- (abbbd37) [Merge pull request #2680 from rspec/update-travis-build-scri…](https://github.com/rspec/rspec-core/commit/abbbd37d3346c580d83ecb96819e29d063d97852)
- (b91a276) [Syncing older travis changes](https://github.com/rspec/rspec-core/commit/b91a2760b7f48354e8247409644391dbef669cf1)
- (1c31fc4) [Avoid bisect command to get stuck (#2669)](https://github.com/rspec/rspec-core/commit/1c31fc4a2a265082b3cec8f6878f258337c0fcd0)
- (968fd99) [Update changelog for #2669](https://github.com/rspec/rspec-core/commit/968fd99b06c57e462ae76db57864c3661c65a750)
- (1120c26) [Merge pull request #2672 from rspec/update-common-plaintext-…](https://github.com/rspec/rspec-core/commit/1120c26f5e77ae11d9d649a86c839fcd78406d3f)
- (b77634d) [Version 3.9.1](https://github.com/rspec/rspec-core/commit/b77634d5ca100b39968e14c94ec00ad660273ec3)
- (4dd320b) [Bump yard to support Ruby 2.7](https://github.com/rspec/rspec-core/commit/4dd320b38bc6376e1355e66995521a961bc9d236)
- (db5b83b) [Merge pull request #2683 from rspec/enable-yard-and-bump-doc](https://github.com/rspec/rspec-core/commit/db5b83b58e57c0517eeb2f3c2e42e979a2ca4682)
- (36c5ad5) [Merge pull request #2687 from rspec/add-around-context-warni…](https://github.com/rspec/rspec-core/commit/36c5ad5a061fb9b5cd1c9aafe423079209fdc2b5)
- (305671f) [Merge pull request #2691 from rspec/update-travis-build-scri…](https://github.com/rspec/rspec-core/commit/305671f64ef78654401c2f860df7feabded51bde)


</details>